packages/opentelemetry-cloud-trace-exporter/src/types.ts

```diff
@@ -166,3 +169,35 @@ export enum Code {
   OK = 0,
   UNKNOWN = 2,
 }
+
+/**
+ * See https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudtrace/v2/trace.proto#L182
```

Comment:
Could you reference a tagged version instead of master?

Reply:
There are no useful tags in there unfortunately. I can do a permarlink from the commit (its super long but oh well)